src/components/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+import parser from 'html-react-parser';
+
+export function Footer() {
+  return (
+    <div className="font-heading text-uppercase text-center text-sm text-gray-500 pt-5 mt-10 mx-auto max-w-screen-xl">
+      <hr className="mx-auto" />
+
+      <div className="py-5">
+        {parser("RASENMAEHER v0.1.0")}
+        
+      </div>
+
+      <hr className="mx-auto w-56" />
+
+      <div className="py-5 text-xs">
+        {parser("Proudly served by PVATK")}
+        <br />
+        {parser("&copy; Puolustusvoimat")}
+        <br />
+        {parser("Yhteys: <p class='underline'>helpdesk@pvarki.fi</p>")}
```

Comment:
What parser provides here that requires it?

Reply:
Well actually, nothing. But guess who used parser in the GuideSection component, and then wrote 2 x 3 guides and supplied all formatting to those written things with html tags?

Reply:
removed this one here though